packages/varnish/src/header/Header.tsx

```diff
@@ -216,6 +212,7 @@ const MenuColumn = styled.div`
  */
 // eslint-disable-next-line import/prefer-default-export
 export const Header = Object.assign(HeaderComponent, {
+  AI2Banner,
```

Comment:
so i had taken this out because to override the default banner in the header, you need a ref to the banner in order to set it here. instead, i propose we have users using a custom banner to pass it in as a prop. it's typically not great practice to override elements with passing around refs, and i think it's cleaner design to have a custom banner passed in as a prop; should we get rid of this since at least right now it is not possible to set this?

Reply:
maybe we rename it DefaultBanner?
i see no reason not to export it 
it is only imported elsewhere if they dont's use the Header component.
maybe im missing something?

Reply:
it is also already exported from AI2Banner itself so do we need it at this layer? 
I am okay keeping it if we feel strongly, just with the caveat that users should not try to use it in the same way they would use 
<Header.AppTagline> Tagline </> 
for example